java/client/src/main/java/glide/api/models/ClusterTransaction.java

```diff
@@ -27,4 +36,69 @@ public class ClusterTransaction extends BaseTransaction<ClusterTransaction> {
     protected ClusterTransaction getThis() {
         return this;
     }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * The <code>sort</code> command can be used to sort elements based on different criteria and
+     * apply transformations on sorted elements.<br>
+     * To store the result into a new key, see {@link #sortStore(String, String, SortClusterOptions)}.
+     *
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @param sortClusterOptions The {@link SortClusterOptions}.
+     * @return Command Response - A <code>Array</code> of sorted elements.
+     */
+    public ClusterTransaction sort(
+            @NonNull String key, @NonNull SortClusterOptions sortClusterOptions) {
+        RedisRequestOuterClass.Command.ArgsArray commandArgs =
+                buildArgs(ArrayUtils.addFirst(sortClusterOptions.toArgs(), key));
+        protobufTransaction.addCommands(buildCommand(Sort, commandArgs));
+        return this;
+    }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * <br>
+     * The <code>sortReadOnly</code> command can be used to sort elements based on different criteria
+     * and apply transformations on sorted elements.<br>
+     * This command is routed depending on the client's {@link ReadFrom} strategy.
```

Comment:
I'm not sure this is true for transactions.  I'd have to check if we use the ReadFrom strategy when all the transaction commands are read-only command.  I'd suggest removing this comment.

Reply:
Leaving as is to remain consistent with python docs.